Samples/OLED1Xplained/OLED1.lpr

```diff
@@ -0,0 +1,152 @@
+program OLED1;
+{
+  This file is part of Pascal Microcontroller Board Framework (MBF)
+  Copyright (c) 2015 -  Michael Ring
+  based on Pascal eXtended Library (PXL)
+  Copyright (c) 2000 - 2015  Yuriy Kotsarenko
+
+  This program is free software: you can redistribute it and/or modify it under the terms of the FPC modified GNU
+  Library General Public License for more
+
+  This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the FPC modified GNU Library General Public
+  License for more details.
+}
+
+{< MBED application shield demo program. }
+
+{$INCLUDE MBF.Config.inc}
+
+{.$define DEBUG}
+
+uses
+  heapmgr, // this is needed for SAMD10, but keep it in place for the others also !!!!
+  MBF.BitHelpers,
+  MBF.__CONTROLLERTYPE__.SystemCore,
+  MBF.__CONTROLLERTYPE__.GPIO,
+  MBF.__CONTROLLERTYPE__.SPI,
+  MBF.Displays.OLED1Xplained;
+
+const
+  {$if defined(has_samd20_xplained_pro) or defined(has_samd21_xplained_pro) or defined(has_samc21_xplained_pro)}
+    PinLED     = LED0;
+    PinButton  = SW0;
+
+    PortSPI    = 5;
+    PortSPIMux = Ord(TPinMux.MuxD);
+    PinMOSIMux = PortSPIMux;
+    PinMISOMux = Ord(TPinMux.None);
+    PinSCKMux  = PortSPIMux;
+
+    PinReset   = EXT3_PIN_10;
+    PinMISO    = TNativePin.None;
+    PinSS      = EXT3_PIN_15;
+    PinMOSI    = EXT3_PIN_16;//PAD[2]
+    PinSCK     = EXT3_PIN_18;//PAD[3]
+    PinA0      = EXT3_PIN_5;
+  {$endif}
+
+var
+  Display:TDisplay;
+  SPI1:TSPI_Registers;
+  TurnedOn:boolean;
+  i:byte;
+
+procedure ToggleLed;
```

Comment:
Why is toggling an LED part of this demo code?

Reply:
For me (and others), to indicate that the board is still alive and kicking !!